server/s3_common.h

```diff
@@ -121,6 +121,7 @@ enum class S3OperationCode {
 enum class S3AuthClientOpType {
   authentication,
   authorization,
+  combo_auth,
```

Comment:
good to have comment for what this combo_auth does

Reply:
Will do